decentraland/unity-renderer
# feat: show all chat notis

## What does this PR change?

`Fixes #3955 `

All the chat messages should be displayed in the chat notification panel, including those you send.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=feat/show-all-chat-notis
2. Send private messages with two different users. Check top notifications aswell (by opening any other chat window)
3. Send messages in ~nearby  with two different users. Check top notifications aswell (by opening any other chat window)
4. Send messages in any #channel  with two different users. Check top notifications aswell (by opening any other chat window)
5. Click the notifications in the chat panel, all should open the corresponding chat window
6. Open the friends window and click on entries to open the private chat window
7. Open the chat list window and click on DM entry to open the private chat window
8. Press ENTER, it should open the last chat conversation. If any was opened, then ~nearby

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
